packs/data/extinction-curse-bestiary.db/hollow-hush.json

```
@@ -1338,11 +1423,58 @@
             },
             "type": "action"
         },
+        {
+            "_id": "joVEQgf7vPuTLuTt",
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "+37 Will Save vs. Emotion Effects",
+            "sort": 1700000,
+            "system": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": ""
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "key": "FlatModifier",
+                        "label": "Vs. Emotion Effects",
```

```suggestion
                        "label": "PF2E.SkillVariant.VsEmotion",
```
No clue why this is in skill variants but it's in there now along with some other save labels so I guess we'll use it regardless.